### ISSUE TITLE
Time Progression Marble Testing Docs Seems Incorrect

### ISSUE THREAD

**Issue by dudewad** [ReactiveX/rxjs#3709]
I think I've discovered an issue with the documentation for [marble testing](https://github.com/ReactiveX/rxjs/blob/master/doc/marble-testing.md#time-progression-syntax) in the time progression syntax section.

The first example contains the following:

```ts
const input = ' -a-b-c|';
const output = '-- 9ms a 9ms b 9ms (c|)';
/*

// Depending on your personal preferences you could also
// use frame dashes to keep vertical aligment with the input
const input = ' -a-b-c|';
const output = '------- 4ms a 9ms b 9ms (c|)';
// or
const output = '-----------a 9ms b 9ms (c|)';

*/

const result = cold(input).pipe(
  concatMap(d => of(d).pipe(
    delay(10)
  ))
);

expectObservable(result).toBe(expected);
```

However, `expected` is not defined in this example. I just want to confirm whether or not `output` should actually be `expected` or vice versa? It appears so.

**Comment by martinsik** [comment]
Seems like this was fixed by #3725.

**Comment by cartant** [comment]
Closing this because:

> Seems like this was fixed by #3725.
